target-platform/target-platform.target

```diff
@@ -47,10 +43,20 @@
 			<unit id="org.eclipse.license.feature.group" version="0.0.0"/>
 		</location>
 		<location includeSource="true" missingManifest="error" type="Maven">
-			<groupId>io.takari.m2e.workspace</groupId>
-			<artifactId>org.eclipse.m2e.workspace.cli</artifactId>
-			<version>0.3.1</version>
-			<type>jar</type>
+			<dependencies>
+				<dependency>
+					<groupId>biz.aQute.bnd</groupId>
+					<artifactId>biz.aQute.bndlib</artifactId>
+					<version>5.3.0</version>
```

Comment:
We need it to be 5.1.2 which was the last release approved ( https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22694 )